src/bot/requirements.txt

```diff
@@ -1,8 +1,192 @@
+entrypoints==0.3
+fbmessenger==6.0.0
+future==0.18.2
+gast==0.2.2
+gevent==1.4.0
+gitdb==4.0.2
+GitPython==3.1.0
+google-auth==1.13.1
+google-auth-oauthlib==0.4.1
+google-pasta==0.2.0
+greenlet==0.4.15
+grpcio==1.28.1
+h11==0.8.1
+h2==3.2.0
+h5py==2.10.0
+hpack==3.0.0
+hstspreload==2020.4.7
+httplib2==0.17.1
+httptools==0.1.1
+httpx==0.9.3
+humanfriendly==8.1
+hyperframe==5.2.0
 idna==2.9
+idna-ssl==1.1.0
+immutables==0.11
+importlib-metadata==1.6.0
+ipykernel==5.2.1
+ipython==7.13.0
+ipython-genutils==0.2.0
+isodate==0.6.0
+jedi==0.17.0
+Jinja2==2.11.2
+jmespath==0.9.5
+joblib==0.14.1
+json5==0.9.4
+jsonpickle==1.3
+jsonschema==3.2.0
+jupyter-client==6.1.3
+jupyter-core==4.6.3
+jupyterlab==2.1.0
+jupyterlab-server==1.1.1
+kafka-python==1.4.7
+Keras-Applications==1.0.8
+Keras-Preprocessing==1.1.0
+kiwisolver==1.2.0
+Mako==1.1.2
+Markdown==3.2.1
+MarkupSafe==1.1.1
+matplotlib==3.1.3
+mattermostwrapper==2.2
+mistune==0.8.4
+multidict==4.7.5
+murmurhash==1.0.2
+nbconvert==5.6.1
+nbformat==5.0.6
+networkx==2.4
+notebook==6.0.3
+numpy==1.18.2
+oauth2client==4.1.3
+oauthlib==3.1.0
+opt-einsum==3.2.0
+packaging==19.0
+pandocfilters==1.4.2
+parsedatetime==2.5
+parso==0.7.0
+pexpect==4.8.0
+pickleshare==0.7.5
+pika==1.1.0
+plac==0.9.6
+preshed==2.0.1
+prometheus-client==0.7.1
+prompt-toolkit==2.0.10
+protobuf==3.11.3
+psycopg2-binary==2.8.5
+ptyprocess==0.6.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycparser==2.20
+pydot==1.4.1
+Pygments==2.6.1
+PyJWT==1.7.1
+pykwalify==1.7.0
+pymongo==3.8.0
+pyparsing==2.4.7
+pyrsistent==0.16.0
+PySocks==1.7.1
 pyTelegramBotAPI==3.6.7
-python-dotenv==0.12.0
+python-crfsuite==0.9.7
+python-dateutil==2.8.1
+python-dotenv==0.13.0
+python-editor==1.0.4
+python-engineio==3.11.2
+python-socketio==4.4.0
+python-telegram-bot==11.1.0
+pytz==2019.3
+PyYAML==5.3.1
+pyzmq==19.0.0
+questionary==1.5.1
+rasa==1.9.5
+rasa-sdk==1.9.0
+rasa-x==0.27.4
+redis==3.4.1
+regex==2020.4.4
 requests==2.23.0
+requests-oauthlib==1.3.0
+requests-toolbelt==0.9.1
+rfc3986==1.4.0
+rocketchat-API==0.6.36
+rsa==4.0
+ruamel.yaml==0.15.100
+s3transfer==0.3.3
+sanic==19.12.2
+Sanic-Cors==0.10.0.post3
+sanic-jwt==1.3.2
+Sanic-Plugins-Framework==0.9.2
+schedule==0.6.0
+scikit-learn==0.22.2.post1
+scipy==1.4.1
+Send2Trash==1.5.0
 six==1.14.0
+sklearn-crfsuite==0.3.6
+slackclient==2.5.0
+smmap==3.0.1
+sniffio==1.1.0
+spacy==2.1.9
+SQLAlchemy==1.3.16
+srsly==1.0.2
+tabulate==0.8.7
+tensorboard==2.1.1
+tensorflow==2.1.0
+tensorflow-addons==0.8.3
+tensorflow-estimator==2.1.0
+tensorflow-hub==0.7.0
+tensorflow-probability==0.7.0
+termcolor==1.1.0
+terminado==0.8.3
+terminaltables==3.1.0
+testpath==0.4.4
+thinc==7.0.8
+tornado==6.0.4
+tqdm==4.31.1
+traitlets==4.3.3
+tsresolve==0.6
+twilio==6.26.3
+typeguard==2.7.1
+typing-extensions==3.7.4.2
+tzlocal==2.0.0
+ujson==1.35
 urllib3==1.25.8
+uvloop==0.14.0
+wasabi==0.6.0
+wcwidth==0.1.9
+webencodings==0.5.1
+webexteamssdk==1.1.1
+websockets==8.1
+Werkzeug==1.0.1
+wrapt==1.12.1
+yarl==1.4.2
+zipp==3.1.0
```

Comment:
Lot of dependencies, revise the requirements list, use pipreqs to generate the requirements.txt based on imports in the your project!

Reply:
Okay , will make the changes